frontend/pages/index.tsx

```diff
@@ -1,85 +1,67 @@
-import Head from "next/head";
-import Link from "next/link";
-import { useState } from "react";
+import { useState, useContext } from "react";
+import { ThemeContext } from "styled-components";
 
-import {Layout, Seo} from '../components'
+import { HelperText, StyledH1, Grid } from "../styles/core-ui";
+import { Layout, Seo } from "../components";
+import FormInput from "../components/FormInput";
+import Button from "../components/Button";
 
 /**
- * Get tutorials 
- * 
+ * Get tutorials
+ *
  * @todo typescript types
  * @returns tutorials
  */
 export const getServerSideProps = async () => {
-    const response = await fetch('http://localhost:3000/api/getTutorial');
-    return {
-        props: {
-            tutorials: await response.json()
-        }
-    }
-}
+  const response = await fetch("http://localhost:3000/api/getTutorial");
+  return {
+    props: {
+      tutorials: await response.json(),
```

Comment:
I think your code formatter is adding commas in the wrong places lol

Reply:
It's just a trailing command supported in objects. It's prettier.